layouts/shortcodes/dev-branch.html

```diff
@@ -0,0 +1,3 @@
+{{- $nextMinorVersion := site.Params.nextMinorVersion }}
+{{- $latestDevBranch := printf "dev-%s" (replace $nextMinorVersion "v" "") }}
```

Comment:
@jimangel , small nit: Do you need two shortcodes? Could you have one shortcode that takes a parameter as the prefix. If no prefix is provided, just return the `nextMinorVersion`, otherwise return the concatenated string? Same type of thing could be done for the release branch.
Wish list: The docs could use a release version parameter without any prefix (no `v`) so that various version strings could be constructed from the parameter.

Reply:
I'm open to any suggestions, I just stole from the existing model:
- https://github.com/kubernetes/website/blob/master/layouts/shortcodes/release-branch.html
- https://github.com/kubernetes/website/blob/master/layouts/shortcodes/latest-version.html

Do you know how we'd pass a parameter into the shortcode or have a document that would tip me off?

/cc @lucperkins for thoughts.

Reply:
Here's some info:
https://gohugo.io/templates/shortcode-templates/#positional-vs-named-parameters